.github/workflows/test-deploy.yml

```diff
@@ -28,10 +28,19 @@ jobs:
       env:
         CI: true
         FIREBASE_API_KEY: ${{ secrets.firebase_api_key }}
+    - name: Deploy Firebase actions
+      run: |
+        npm install --prefix ./functions/ ./functions/
+        npm install -g firebase-tools
+        firebase use --token "$FIREBASE_CI_KEY" youtwobe
+        firebase deploy --token "$FIREBASE_CI_KEY" --only functions
```

Comment:
Note that if conflictions between Firebase cloud functions becomes an issue the `firebase deploy` command does have a `--force` option which can be added.  This will overwrite previous conflictions functions with the new functions attempting to be registered. If a conflictions occurs, the deployment will be cancelled, so it's something useful to know.